dbms/src/Flash/Mpp/ExchangeReceiver.cpp

```diff
@@ -746,6 +746,9 @@ void ExchangeReceiverBase<RPCContext>::connectionDone(
         }
         copy_live_conn = --live_connections;
     }
+
+    cv.notify_all();
```

Comment:
only needs to be called when `meet_error || copy_live_conn == 0` is true.

Reply:
```
    if (meet_error || copy_live_conn == 0)
    {
        finishAllMsgChannels();
        cv.notify_all();
    }
        
```

Reply:
Move to L765

Reply:
> Move to L765

okk